### PR TITLE
Add library `jquery.inputmask`

### PR DESCRIPTION
**Pull request by chrisaljoudi**
Active; [1,273 stars on GitHub](https://github.com/RobinHerbots/jquery.inputmask).

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Would you please also add auto-update config as our document said? Thanks.

**Comment by chrisaljoudi** [comment]
@PeterDaveHello done — sorry about that! Didn't notice.

**Comment by PeterDaveHello** [comment]
Don't we need files under `inputmask/` folder?

File: ajax/libs/jquery.inputmask/package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "name": "jquery.inputmask",
+  "version": "3.1.62",
+  "description": "jquery.inputmask is a jquery plugin which create an input mask.",
+  "main": "./dist/inputmask/jquery.inputmask.js",
+  "npmName": "jquery.inputmask",
+  "npmFileMap": [
+    {
+      "basePath": "/dist/",
+      "files": [
+        "jquery.inputmask*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.inputmask/package.json]: What about `**` or `jquery.inputmask**` here?

**Comment by chrisaljoudi** [comment]
@PeterDaveHello those are basically the modules/components that make up the `bundle` file (which is what we currently have).

I figured since it wasn't done with jQuery/etc., it needn't be done here.

**Comment by PeterDaveHello** [comment]
Okay, fine.

**Pull request merged**